### PR TITLE
Handle C++11 defaulted constructor.

### PR DESCRIPTION
**Pull request by ratiotile**
See issue #88

### CODE REVIEW

File: cparse.nim [hunk 1]
```diff
@@ -2172,6 +2172,11 @@ proc parseConstructor(p: var Parser, pragmas: PNode, isDestructor: bool;
     let body = compoundStatement(p)
     if pfKeepBodies in p.options.flags:
       result.sons[bodyPos] = body
+  of pxAsgn:
+    # '= default;' C++11 defaulted constructor
+    getTok(p)
+    eat(p, pxSymbol)
```
Comment (Araq) [inline cparse.nim]: Please check for 'default' here too.

Comment (ratiotile) [inline cparse.nim]: Would replacing 2178: `eat(p, pxSymbol)` with `eat(p, "default")` do it?

Comment (ratiotile) [inline cparse.nim]: with the appropriate overload of eat.

Comment (Araq) [inline cparse.nim]: Already merged and adapted. But in principle, sure, seems nice to have an overloaded 'eat'. ;-)

**Pull request merged**